src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private String orderDate;
```

Comment:
Можливо варто мати поле типу LocalDate або LocalDateTime?
```suggestion
    private LocalDate/LocalDateTime orderDate;
```